scss/_footer.scss

```diff
@@ -0,0 +1,72 @@
+body .container {
+  padding: 1em 1em 0em 1em !important ;
+}
+
+#s-lc-public-footer {
+  background-color: $text_black;
+  border-top: none;
+  margin: 0 auto;
+  font-size: 100%;
+  color: $text_footer_gray;
+  padding: 45px 0 0 0;
+  line-height: 1.75;
+}
+
+#s-lc-public-footer-admin-links,
+#s-lc-public-footer-brand,
+#s-lc-footer-support-link,
+#s-lc-language,
+.public-footer-container {
```

Comment:
@laurxnemeth I noticed that the 'screen-readers' link on space pages isn't picking up these styles (see: http://localhost:8080/reserve/grad-collab). You may want to add a `.s-lc-public-footer-print-link` here as well or also possibly a `footer` or `footer#s-lc-public-footer` and see how much that gets you over the individual classes/ids.